### PR TITLE
nix: Apply some nixfmt suggestions and fix macOS status-go buildInputs

### PR DESCRIPTION
**Pull request by pedropombeiro**
[comment]: # (Please replace ... with your information. Remove < and >)
[comment]: # (To auto-close issue on merge, please insert the related issue number after # i.e fixes #566)

### Summary

[comment]: # (Summarise the problem and how the pull request solves it)
This PR fixes some minor formatting issues in Nix files (looking at nixfmt suggestions) and also the way that `status-go.buildInputs` was being included for macOS.

### Review notes

status: ready

### CODE REVIEW

File: nix/mobile/android/android-env.nix [hunk 1]
```diff
@@ -25,7 +25,8 @@ let
     ndkVersion = "19.2.5345600";
     useGoogleAPIs = false;
     useGoogleTVAddOns = false;
-    includeExtras = [ "extras;android;m2repository" "extras;google;m2repository" ];
+    includeExtras =
+      [ "extras;android;m2repository" "extras;google;m2repository" ];
```
Comment (jakubgs) [inline nix/mobile/android/android-env.nix]: Really dislike this format, looks like a syntax error to me. This formatter looks pretty inconsistet in how it handles arrays, for example look at `nix/desktop/macos/default.nix`.

**Review by jakubgs — APPROVED** [review]

File: nix/mobile/android/android-env.nix (hunk 1 above)
Comment (pedropombeiro) [reply nix/mobile/android/android-env.nix]: Yeah, I guess it is trying to make lines as short as possible. I agree with you that it still needs quite a bit of optimization (at the very least be able to turn on/off certain rules). I'll rollback these particular changes for now.

**Review by flexsurfer — APPROVED** [review]

**Comment by pedropombeiro** [comment]
@churik is it normal that 9 e2e tests are failing?

**Comment by churik** [comment]
@PombeirP let me relaunch 2 tests, rest of e2e are failed due to https://github.com/status-im/status-react/issues/8645

**Comment by churik** [comment]
These tests https://github.com/status-im/status-react/pull/8656#issuecomment-516022681 are also false failures, so ready to go :)

**Pull request merged**